nrf_802154/doc/antenna_diversity.rst

```diff
@@ -184,7 +184,7 @@ The highest detected energy across both antennas is reported to the driver.
 
 .. note::
    In manual mode, the module performs energy detection on a single antenna.
-   The application specifies the antenna on which to perfom the energy detection calling the :c:func:`nrf_802154_antenna_diversity_rx_antenna_set` function.
+   The application specifies the antenna on which the energy detection must be performed by calling the :c:func:`nrf_802154_antenna_diversity_rx_antenna_set` function.

```

Comment:
I think 'shall be performed by' is a better choice of words. Energy detection is a functionality, not a necessity.